File: AndroidApp/MultiSportTrainer-Mobile/app/src/main/java/com/example/multisporttrainer/models/UpdateUserRequest.java
package com.example.multisporttrainer.models;

public class UpdateUserRequest {

    private String fullName;
    private String email;
    private String dateOfBirth;
    private String role;
    private String sportFocus;

    public UpdateUserRequest(String fullName, String email, String dateOfBirth, String role, String sportFocus) {
        this.fullName = fullName;
        this.email = email;
        this.dateOfBirth = dateOfBirth;
        this.role = role;
        this.sportFocus = sportFocus;
    }
}

File: Backend/MultiSportTrainerAPI/MultiSportTrainerAPI/DTOs/UpdateUserRequest.cs
namespace MultiSportTrainerAPI.DTOs
{
    public class UpdateUserRequest
    {
        public string FullName { get; set; } = string.Empty;

        public string Email { get; set; } = string.Empty;

        public DateTime? DateOfBirth { get; set; }

        public string Role { get; set; } = "Player";

        public string? SportFocus { get; set; }
    }
}